Initial troubleshooting & issue templates (#579)
brimsec/brim: TROUBLESHOOTING.md

therefore be extracted from the original pcap as if it were a separate flow).

These represent what we expect may be the majority of observed differences in
flow handling. However, through side-by-side study of Wireshark next to Zeek,
we've uncovered the following additional corner cases that may also yield
differences:

|**Issue**|**Description**|
|-|-|
|[zeek/846](https://github.com/zeek/zeek/issues/846)|"ts+duration" for TCP flow is missing more than just ACK|
|[zeek/847](https://github.com/zeek/zeek/issues/847)|Zeek creates many conn records for DNS despite `udp_inactivity_timeout=0`|
|[zeek/862](https://github.com/zeek/zeek/issues/862)|Retransmissions of unsuccessful close attempt keep getting flagged as new connections|
|[zeek/864](https://github.com/zeek/zeek/issues/864)|Multi-section pcapng with overlapping timestamps creates excess `conn` events|

If you find yourself running into these issues or others of a similar nature,
please reach out to us on our [public Slack](https://join.slack.com/t/brimsec/shared_invite/zt-cy34xoxg-hZiTKUT~1KdGjlaBIuUUdg)
or email us at support@brimsecurity.com so we can chat further.

# Gathering Info

> Before [opening an issue](#opening-an-issue), or during debug on a
> previously-opened issue, there's detail you can gather that may help us more

Reviewer: ```suggestion
Before [opening an issue](#opening-an-issue), or while debugging a
previously-opened issue, there's detail you can gather that may help us more
```
